server/pipeline/topic.go

```diff
@@ -33,9 +34,14 @@ func publishToTopic(pipeline *model.Pipeline, repo *model.Repo) {
 	}
 	pipelineCopy := *pipeline
 
-	message.Data, _ = json.Marshal(model.Event{
+	var err error
+	message.Data, err = json.Marshal(model.Event{
```

Comment:
```suggestion
	message.Data, err := json.Marshal(model.Event{
```

Should work as well?